python/nano/src/bigdl/nano/utils/inference/pytorch/model_utils.py

```diff
@@ -32,6 +36,95 @@ def get_forward_args(model):
     return forward_args
 
 
+def get_forward_defaults(model):
+    '''
+    This function is to get all the defaults
+    It will return a list of default values
+    '''
+    forward_defaults = inspect.getfullargspec(model.forward).defaults
+    if isinstance(model, LightningModule):
+        if not isinstance(model, AcceleratedLightningModule):
+            # forward param list for compiled model
+            forward_defaults = get_forward_defaults(model.model)
+    return forward_defaults
+
+
+def get_forward_annotations(model):
```

Comment:
Maybe more comments to explain the three functions, it's a little hard to understand now.

Reply:
Done